tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -40,6 +40,11 @@ def stop_pfcwd(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     logger.info("--- Stop Pfcwd --")
     duthost.command("pfcwd stop")
 
+    yield
```

Comment:
If the stop/start apply to all the cases, could we include it in the fixture at the module level? thanks

Reply:
adjusted the scope to module @lipxu